project_timesheet_time_control/wizards/hr_timesheet_switch_view.xml

```diff
@@ -23,10 +23,10 @@
                         <field name="project_id" required="1" />
                         <field name="task_id" />
                         <field name="company_id" groups="base.group_multi_company" />
+                        <field name="company_id" invisible="1" />
                     </group>
-                    <group
+                    <div
                         name="messages"
-                        string="Previous timer"
```

Comment:
You are removing a text that was appearing here (`Previous timer`). Please put the equivalent one, but you can't remove it.

Reply:
Changes corrected, thanks!

Reply:
It was not correct. The label was still not shown, but even more, it has problems not showing in the same line the rest of the fields. Now it is.